includes/utils.hpp

```diff
@@ -0,0 +1,10 @@
+#ifndef UTILS_HPP
+# define UTILS_HPP
+
```

Comment:
I would put here all the dependencies/libraries outside the project and place `# include "utils.hpp"` both `server.hpp` and `socket.hpp` - I feel it would be easier to track what's already there and it should avoid mistakes

Reply:
Ok done!